### PR TITLE
Another existing proc removed.

### PR DESCRIPTION
**Pull request by eswordthecat**

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -2215,45 +2215,59 @@
...
-				for(var/mob/living/carbon/human/H in mob_list)
-					var/turf/loc = find_loc(H)
-					var/security = 0
-					if(loc.z > 1 || prisonwarped.Find(H))
-//don't warp them if they aren't ready or are already there
+
+				var/security
+
+				for (var/mob/living/carbon/human/H in mob_list)
+					if (H in prisonwarped) // don't warp them if they aren't ready or are already there
```
Comment (ComicIronic) [inline code/modules/admin/topic.dm]: Shouldn't you keep the z check?

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -2215,45 +2215,59 @@
...
+
+						for (var/A in id.access)
+							if (A == access_security)
+								security = TRUE
+								break
+
+					if (!security)
+						// strip their stuff before they teleport into a cell :downs:
+						for (var/obj/item/weapon/W in H)
+							if (istype(W, /datum/organ/external)) //don't strip organs
```
Comment (ComicIronic) [inline code/modules/admin/topic.dm]: How does that even happen?

Comment (eswordthecat) [inline code/modules/admin/topic.dm]: Dunno, ask the olde codars.

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (eswordthecat) [inline code/modules/admin/topic.dm]: That would skip people in mining.

Comment (eswordthecat) [inline code/modules/admin/topic.dm]: And research asteroid.

Comment (ComicIronic) [inline code/modules/admin/topic.dm]: I think that was the point of the z_level check.

Comment (eswordthecat) [inline code/modules/admin/topic.dm]: But the admin command says, teleported all players to the prison station.

File: code/modules/admin/topic.dm (hunk 2 above)
Comment (ComicIronic) [inline code/modules/admin/topic.dm]: Change it from in H to in H.get_all_slots()

**Comment by eswordthecat** [comment]
Hellooooooooo

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 3]
```diff
@@ -17,7 +17,7 @@
 			H << "<span class='warning'>You are unable to equip that.</span>"
 
 /mob/living/carbon/human/get_all_slots()
-	return (get_head_slots() || get_body_slots()) //the || is list unification
+	. = get_head_slots() | get_body_slots()
```
Comment (ComicIronic) [inline code/modules/mob/living/carbon/human/inventory.dm]: literally why

**Pull request merged**